### PR TITLE
secure data mgmt API with auth key

### PR DESCRIPTION
**Pull request by DominikPinsel**

### CODE REVIEW

File: edc-controlplane/README.md [hunk 1]
```diff
@@ -458,7 +458,7 @@ curl -X POST "$__connectorUrl/$__dataMgmtPath/policies" --header "$__apiKey: $__
 Get Contract Definition
 
 ```bash
-curl -X GET "$__connectorUrl/$__dataMgmtPath/policies/$__policyId" --header "$__apiKey: $__apiKeyValue" --header "Content-Type: application/json" | jq
+curl -X GET "$__connectorUrl/$__dataMg0mtPath/policies/$__policyId" --header "$__apiKey: $__apiKeyValue" --header "Content-Type: application/json" | jq
```
Comment (florianrusch-zf) [inline edc-controlplane/README.md:461]: Typo?

```suggestion
curl -X GET "$__connectorUrl/$__dataMgmtPath/policies/$__policyId" --header "$__apiKey: $__apiKeyValue" --header "Content-Type: application/json" | jq
```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]
We need to document how to set the token within the edc configuration

File: edc-controlplane/README.md (hunk 1 above)
Comment (DominikPinsel) [reply edc-controlplane/README.md]: Ah yes :) Thanks for the review

**Review by florianrusch-zf — APPROVED** [review]

**Comment by florianrusch-zf** [comment]
Tested with api-wrapper 0.0.3 🎉   works!!

**Pull request closed without merging**